src/smartcontracts/transactionPayloadBuilders.ts

```diff
@@ -1,11 +1,10 @@
+import { WasmVirtualMachine } from "../constants";
```

Comment:
Should be fine, we can consider that this isn't a breaking change.

Reply:
I don't see why this could be breaking.

Reply:
Being a publicly exported element, a movement can affect clients that use fully qualified imports (in this context, we can simply think of this as a non-breaking change, though).